students/vkis/lesson03/list_lab.py

```diff
@@ -0,0 +1,73 @@
+select = input("\nEnter which Series (1,2,3,4) to run: ")
+while select != "1" and select != "2" and select != "3" and select != "4":		# is there away to run a variable check against a list of conditions without repeating like this?
+	select = input("Incorrect selection enter Series: 1,2,3, or 4 to run: ")
+
+# Series 1
+if select == "1":
+	print()
+	mylist = ["Apples","Pears","Oranges","Peaches"]
+	print("[OUT]: mylist = ", mylist)
+	print()
```

Comment:
instead of these `print()` statements to create a new line, just put them in your output print:
```
print("\n[OUT]: mylist = \n", mylist)
```